### PR TITLE
Canvas: Docs for 9.2

### LINKED ISSUE #52826: Canvas: Draft initial beta documentation
Draft up the initial documentation for canvas panel. These should be pretty high level and clearly lay out some solid use cases / the "why" behind canvas vs. the small details (as those will probably change quickly over time for the foreseeable future)

### PR DESCRIPTION
**Pull request by nmarrs**
**What this PR does / why we need it**:
Update docs for v9.2 canvas panel beta release

**Which issue(s) this PR fixes**:

Fixes #52826

### CODE REVIEW

File: docs/sources/visualizations/canvas.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - documentation
+title: Canvas
+weight: 600
+---
+
+# Canvas
+
+The Canvas panel visualization allows you to ...
+
+TODO: add gif from what's new
```
Comment (drew08t) [inline docs/sources/visualizations/canvas.md:19]: Are GIFs supported? I wasn't able to get them to work previously.

File: docs/sources/whatsnew/whats-new-in-v9-2.md [hunk 2]
```diff
@@ -33,6 +33,16 @@ For details, see [GitHub issue #55005](https://github.com/grafana/grafana/issues
 
 {{< figure src="/static/img/docs/panels/panel-help-9-2.gif" max-width="750px" caption="Retrieving a panel's query response data and panel settings" >}}
 
+## Canvas panel
+
+_Available in beta in Grafana Open Source._
+
+Introducing the Canvas panel, a new panel that combines the power of Grafana with the flexibility of custom elements.
+Canvas is a new panel type that allows you to create custom visualizations and overlay data.
+Canvas is a great way to create custom visualizations that are not possible with standard Grafana panels.
```
Comment (gguillotte-grafana) [inline docs/sources/whatsnew/whats-new-in-v9-2.md:42]:
```suggestion
Canvas visualizations are extensible form-built panels that allow you to explicitly place elements within static and dynamic layouts. This empowers you to design custom visualizations and overlay data in ways that aren't possible with standard Grafana panels, all within Grafana's UI. If you've used popular UI and web design tools, then designing Canvas panels will feel very familiar.

For example, you can place image layers and then overlay text that's updated by Grafana data sources, and display or hide icons conditionally based on data.

We've planned additional features and design elements for future releases to make Canvas panels even more powerful tools for creating custom, interactive, data-driven visualizations.
```

**Review by gguillotte-grafana — COMMENTED** [review]
Suggested additions to the What's New text.

File: docs/sources/visualizations/canvas.md (hunk 1 above)
Comment (gguillotte-grafana) [reply docs/sources/visualizations/canvas.md:19]: GIFs are supported in the documentation, but either static screenshots or video (even silent video) are preferred.

Comment (nmarrs) [reply docs/sources/visualizations/canvas.md:19]: It appears that they are based on gif for [this section](https://grafana.com/docs/grafana/next/whatsnew/whats-new-in-v9-2/#panel-help-menu) in current 9.2 what's new docs

File: docs/sources/whatsnew/whats-new-in-v9-2.md [hunk 3]
```diff
@@ -33,6 +33,19 @@ For details, see [GitHub issue #55005](https://github.com/grafana/grafana/issues
...
+
+_Available in beta in Grafana Open Source._
+
+Introducing the Canvas panel, a new panel that combines the power of Grafana with the flexibility of custom elements.
+Canvas visualizations are extensible form-built panels that allow you to explicitly place elements within static and dynamic layouts. This empowers you to design custom visualizations and overlay data in ways that aren't possible with standard Grafana panels, all within Grafana's UI. If you've used popular UI and web design tools, then designing Canvas panels will feel very familiar.
+
+For example, you can place image layers and then overlay text that's updated by Grafana data sources, and display icons that can change color conditionally based on data.
+
+We've planned additional features and design elements for future releases to make Canvas panels even more powerful tools for creating custom, interactive, data-driven visualizations.
+
```
Comment (nmarrs) [inline docs/sources/whatsnew/whats-new-in-v9-2.md:46]: TODO: add link to canvas page in docs here?

**Comment by nmarrs** [comment]
Movies / images added in [https://github.com/grafana/website/pull/10260](https://github.com/grafana/website/pull/10260)

File: docs/sources/whatsnew/whats-new-in-v9-2.md [hunk 4]
```diff
@@ -33,6 +33,19 @@ For details, see [GitHub issue #55005](https://github.com/grafana/grafana/issues
...
+## Canvas panel
+
+_Available in beta in Grafana Open Source._
+
+Introducing the Canvas panel, a new panel that combines the power of Grafana with the flexibility of custom elements.
+Canvas visualizations are extensible form-built panels that allow you to explicitly place elements within static and dynamic layouts. This empowers you to design custom visualizations and overlay data in ways that aren't possible with standard Grafana panels, all within Grafana's UI. If you've used popular UI and web design tools, then designing Canvas panels will feel very familiar.
+
+For example, you can place image layers and then overlay text that's updated by Grafana data sources, and display icons that can change color conditionally based on data.
+
+We've planned additional features and design elements for future releases to make Canvas panels even more powerful tools for creating custom, interactive, data-driven visualizations. To learn more about the Geomap panel, see the [documentation]({{< relref "../visualizations/canvas/" >}}).
```
Comment (ryantxu) [inline docs/sources/whatsnew/whats-new-in-v9-2.md:45]:
```suggestion
We've planned additional features and design elements for future releases to make Canvas panels even more powerful tools for creating custom, interactive, data-driven visualizations. To learn more about the Canvas panel, see the [documentation]({{< relref "../visualizations/canvas/" >}}).
```

File: docs/sources/visualizations/canvas.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  - canvas
+  - panel
+  - documentation
+title: Canvas
+weight: 600
+---
+
+# Canvas
+
+Introducing the Canvas panel, a new panel that combines the power of Grafana with the flexibility of custom elements. Canvas visualizations are extensible form-built panels that allow you to explicitly place elements within static and dynamic layouts. This empowers you to design custom visualizations and overlay data in ways that aren't possible with standard Grafana panels, all within Grafana's UI. If you've used popular UI and web design tools, then designing Canvas panels will feel very familiar.
```
Comment (ryantxu) [inline docs/sources/visualizations/canvas.md:17]: This page is not an announcement that it exists, it is just the docs.  Maybe start as:

"Canvas panel is..."

File: docs/sources/visualizations/canvas.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+  - documentation
+title: Canvas
+weight: 600
+---
+
+# Canvas
+
+Canvas is a new panel that combines the power of Grafana with the flexibility of custom elements. Canvas visualizations are extensible form-built panels that allow you to explicitly place elements within static and dynamic layouts. This empowers you to design custom visualizations and overlay data in ways that aren't possible with standard Grafana panels, all within Grafana's UI. If you've used popular UI and web design tools, then designing Canvas panels will feel very familiar.
+
+{{< figure src="/static/img/docs/canvas-panel/canvas-beta-overview-9-2-0.mov" max-width="750px" caption="Canvas panel beta overview" >}}
```
Comment (ryantxu) [inline docs/sources/visualizations/canvas.md:19]: I think something funky with the .mov files. -- should they be something else?

When I try loading locally I see: 
image

but other images elsewhere work 🤔

Comment (ArturWierzbicki) [reply docs/sources/visualizations/canvas.md]: replaced the `figure` tag with `video-embed` - it does work now

**Review by gguillotte-grafana — APPROVED** [review]

**Pull request merged**